_cobalt.yml

```
@@ -0,0 +1,6 @@
+site:
+  title: Wasmtime
+  description: A fast and secure runtime for WebAssembly
+  base_url: http://wasmtime.dev
```

should this be https?
```suggestion
  base_url: https://wasmtime.dev
```